media-libs/libsdl2/libsdl2-2.28.5.ebuild

```diff
@@ -205,10 +207,27 @@ multilib_src_configure() {
 	)
 
 	ECONF_SOURCE="${S}" econf "${myeconfargs[@]}"
+
+	# Most of these workarounds courtesy Debian
+	# https://salsa.debian.org/sdl-team/libsdl2/-/blob/debian/latest/debian/rules
+	local mytestargs=(
+		--x-includes="/usr/include"
+		--x-libraries="/usr/$(get_libdir)"
+		SDL_CFLAGS="-I${S}/include"
+		SDL_LIBS="-L${BUILD_DIR}/build/.libs -lSDL2"
+		ac_cv_lib_SDL2_ttf_TTF_Init=no
+		CFLAGS="${CPPFLAGS} ${CFLAGS} ${LDFLAGS}"
+	)
+
+	use test && \
+		mkdir "${BUILD_DIR}/test" && \
+		cd "${BUILD_DIR}/test" && \
+		ECONF_SOURCE="${S}/test" econf "${mytestargs[@]}"
```

Comment:
This really asks for an `if`, also it doesn't handle errors correctly.

Reply:
Done